src/mlpack/methods/hoeffding_trees/hoeffding_tree.hpp

```diff
@@ -87,6 +87,8 @@ class HoeffdingTree
    *      in batch training mode.
    * @param minSamples If the node has seen this many points or fewer, no split
    *      will be allowed.
+   * @param categoricalSplitIn - Categorical Split into.
+   * @param numericSplitIn - Numeric Split into.
```

Comment:
```suggestion
   * @param categoricalSplitIn Optional instantiated categorical split object.
   * @param numericSplitIn Optional instantiated numeric split object.
```
I think the `-` isn't necessary?  Anyway, I clarified the documentation for those two parameters a little (which look pretty head-scratchey, but luckily I remember when I wrote this...).